engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/impl/JobRestServiceImpl.java

```diff
@@ -16,7 +16,12 @@
  */
 package org.camunda.bpm.engine.rest.impl;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+
```

Comment:
According to our [coding guidelines](https://github.com/camunda/camunda-bpm-platform/wiki/Coding-Style-Java#imports), all non-static imports should be in one block. Ideally, they are sorted alphabetically. Feel free to apply this to all other classes you are changing here as well, in case imports are adjusted.